WISE-Community/WISE
# Newly-added private note now show up in the personal space in the notebook

This is the fix for #1239.

How to test:
1. In preview, add a private note.
2. See that the private note shows up in the "personal" space in the notebook.
